tmt/steps/provision/__init__.py

```diff
@@ -479,6 +407,214 @@ def _export_environment(environment):
             return ""
         return f'export {" ".join(tmt.utils.shell_variables(environment))}; '
 
+    def ansible(self, playbook, extra_args=None):
+        """ Prepare guest using ansible playbook """
+
+        raise NotImplemented
```

Comment:
Should be `raise NotImplementedError` instead, right?

Reply:
Correct, fixed in 001ecf9ad06e8167e1afa260e1f57e25fe213b24.